plugins/LilyPad/Config.cpp

```diff
@@ -30,6 +30,14 @@
 // Needed to know if raw input is available.  It requires XP or higher.
 #include "RawInput.h"
 
+
+const wchar_t *padTypes[] = {
+	L"Unplugged",
+	L"Dualshock 2",
+	L"Guitar" ,
+	L"pop'n controller"
```

Comment:
I really think this should be ``Pop'n Music controller`` as that's how the controller is officially stated on the wiki's.